pkg/webui/console/views/device-add/repository/device-selection/index.js

```diff
@@ -58,6 +59,7 @@ const Selection = props => {
         name="version_ids.brand_id"
         required
         onChange={onBrandChange}
+        glossaryId={glossaryIds.DEVICE_BRAND}
```

Comment:
Currently, implementation of field tooltip assumes that for each glossary ID there is a matching entry in https://www.thethingsindustries.com/docs/reference/glossary. As far as I can see, this and some other entries are missing. We either have to extend the list of glossary items @benolayinka or remove glossary ids that are missing in the glossary list, or change the implementation of field tooltips

Reply:
Does every tooltip need a glossaryId? For now, I can add the items, but in the future there may be a case where we would like to add tooltips for items with no glossary element.

Reply:
Glossary IDs are optional. But let's try to add them whenever we have a relevant entry in the glossary. Not that we also have the possibility to link arbitrary links of the documentation to a tooltip.